docs/make.jl

```diff
@@ -11,11 +11,12 @@ makedocs(
     authors = "The developers of SBML.jl",
     linkcheck = !("skiplinks" in ARGS),
     pages = ["Home" => "index.md", "Reference" => "functions.md"],
+    strict = [:missing_docs, :cross_references],
 )
 
 deploydocs(
     repo = "github.com/LCSB-BioCore/SBML.jl.git",
     target = "build",
     branch = "gh-pages",
-    push_preview = true,
+    push_preview = false,
```

Comment:
Out of curiosity, why?

Reply:
I guess we don't really need the PR doc to be deployed.

Not that it wouldn't be useful at times, but for most occasions the check from PR is perfectly sufficient, and there's no reason to keep the preview docs rotting in gh-pages forever :D

Reply:
You can automatically delete the directory after the PR is closed, see for example https://github.com/JuliaParallel/MPI.jl/blob/f8e4749c1ce8141458d0a88664044ec64da357ba/.github/workflows/PreviewCleanup.yml

Reply:
uuh cool. Writing that down. :D